roles/entitlement_deploy/tasks/main.yml

```diff
@@ -36,7 +52,8 @@
   - name: "Deploy the repo CA from file '{{ entitlement_mc_rhsm_ca }}'"
     shell:
       set -o pipefail;
-      python3 "{{ entitlement_py_apply }}"
-              "{{ entitlement_mc_rhsm_ca }}" BASE64_ENCODED_RHSM_CA_FILE "{{ entitlement_repo_ca }}"
+      cat "{{ entitlement_mc_rhsm_ca }}" 
```

Comment:
I used `cat` instead of `<` just because it allowed me to align the `python3` calls nicely

Reply:
would it be complicated to use Ansible template instead of the Python script?

just wondering what would be simpler/easier to read

Reply:
answered here https://github.com/openshift-psap/ci-artifacts/pull/182#issuecomment-884958996